mlflow/server/handlers.py

```diff
@@ -1435,11 +1435,24 @@ def gateway_proxy_handler():
             message="Deployments proxy request must specify a gateway_path.",
             error_code=INVALID_PARAMETER_VALUE,
         )
-    request_type = request.method
-    json_data = args.get("json_data", None)
-
-    response = requests.request(request_type, f"{target_uri}/{gateway_path}", json=json_data)
+    method = request.method
+    # validate gateway_path
+    if method == "GET":
```

Comment:
nit: shall we separate this validation logic into a function for better logic flow?